integration-tests/src/test/java/org/web3j/protocol/scenarios/FastRawTransactionManagerIT.java

```diff
@@ -96,6 +97,24 @@ public void testTransactionPolling() throws Exception {
         assertTrue(transactionReceipts.isEmpty());
     }
 
+    @Test
+    public void testTransactionResetNonce() throws Exception {
+        FastRawTransactionManager transactionManager =
+                new FastRawTransactionManager(
+                        web3j,
+                        ALICE,
+                        new PollingTransactionReceiptProcessor(
+                                web3j, POLLING_FREQUENCY, DEFAULT_POLLING_ATTEMPTS_PER_TX_HASH));
+
+        Transfer transfer = new Transfer(web3j, transactionManager);
+        BigInteger gasPrice = transfer.requestCurrentGasPrice();
+
+        createTransaction(transfer, gasPrice).send();
+        createTransaction(transfer, gasPrice).send();
+        transactionManager.resetNonce();
```

Comment:
Add check after reset what is the value of nonce

Reply:
Done!